agir/events/views/api_views.py

```diff
@@ -199,6 +187,18 @@ class EventManagementPermissions(GlobalOrObjectPermissions):
     }
 
 
+class EventDetailAPIView(RetrieveAPIView):
+    permission_classes = EventManagementPermissions
+    serializer_class = EventSerializer
+    queryset = Event.objects.all()
+
+
+class EventDetailAdvancedAPIView(RetrieveAPIView):
+    permission_classes = EventManagementPermissions
+    serializer_class = EventAdvancedSerializer
+    queryset = Event.objects.all()
```

Comment:
Ça ne fonctionnera pas normalement, car il n'y a pas de GET dans la classe EventManagementPermissions.
Et en plus pour EventDetailAPIView, on ne veut pas restreindre uniquement aux organisateurs.
```suggestion
class EventDetailPermissions(GlobalOrObjectPermissions):
    perms_map = {
        "GET": [],
    }
    object_perms_map = {
        "GET": ["events.view_event"],
    }

class EventDetailAdvancedPermissions(GlobalOrObjectPermissions):
    perms_map = {
        "GET": [],
    }
    object_perms_map = {
        "GET": ["events.change_event"],
    }


class EventDetailAPIView(RetrieveAPIView):
    permission_classes = (EventDetailPermissions,)
    serializer_class = EventSerializer
    queryset = Event.objects.all()


class EventDetailAdvancedAPIView(RetrieveAPIView):
    permission_classes = (EventDetailAdvancedPermissions,)
    serializer_class = EventAdvancedSerializer
    queryset = Event.objects.all()
```